component/src/test/java/org/wso2/extension/siddhi/io/googlepubsub/sink/TestCaseOfGooglePubSubSink.java

```diff
@@ -0,0 +1,266 @@
+/*
+ *  Copyright (c) 2019  WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ *  WSO2 Inc. licenses this file to you under the Apache License,
+ *  Version 2.0 (the "License"); you may not use this file except
+ *  in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *  http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing,
+ *  software distributed under the License is distributed on an
+ *  "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ *  KIND, either express or implied.  See the License for the
+ *  specific language governing permissions and limitations
+ *  under the License.
+ *
+ */
+package org.wso2.extension.siddhi.io.googlepubsub.sink;
+
+import org.apache.log4j.Logger;
+import org.testng.AssertJUnit;
+import org.testng.annotations.BeforeMethod;
+import org.testng.annotations.Test;
+import org.wso2.extension.siddhi.io.googlepubsub.util.ResultContainer;
+import org.wso2.siddhi.core.SiddhiAppRuntime;
+import org.wso2.siddhi.core.SiddhiManager;
+import org.wso2.siddhi.core.event.Event;
+import org.wso2.siddhi.core.exception.SiddhiAppCreationException;
+import org.wso2.siddhi.core.stream.input.InputHandler;
+import org.wso2.siddhi.core.stream.output.StreamCallback;;
+import org.wso2.siddhi.query.api.exception.SiddhiAppValidationException;
+
+public class TestCaseOfGooglePubSubSink {
+
+    private static Logger log = Logger.getLogger(TestCaseOfGooglePubSubSink.class);
+    private volatile int count;
+    private volatile boolean eventArrived;
+
+    @BeforeMethod
+    public void initBeforeMethod() {
+
```

Comment:
Please remove additional space

Reply:
when we run wso2 code-style that line added automatically.